src/cheat_gui.cpp

```diff
@@ -249,7 +249,7 @@ struct CheatWindow : Window {
 
 		bool rtl = _current_text_dir == TD_RTL;
 		uint button_left = rtl ? ir.right - SETTING_BUTTON_WIDTH : ir.left;
-		uint text_left   = ir.left + (rtl ? 0 : WidgetDimensions::scaled.hsep_wide + SETTING_BUTTON_WIDTH);
+		uint text_left 	 = ir.left + (rtl ? 0 : WidgetDimensions::scaled.hsep_wide + SETTING_BUTTON_WIDTH);
```

Comment:
Sneaky space snuck in here somehow. 😉

Reply:
Did not realise there was a tab there, sorry. Should be fixed now.